runner/src/main/scala/stress/utils/Scenarios.scala

```diff
@@ -104,6 +104,23 @@ object Scenarios {
       )
   }
 
+  def bulkOpsScenario(repeats: Int): ScenarioBuilder = {
+    scenario("(Bulk) Ops person")
+      .repeat(repeats)(
+        exec(Releases.getRandomTreeRelease)
```

Comment:
Just take a look on any other scenario, it works the same way.. I don't like the idea, but I am following the approach here. Should we create a ticket to fix it?